virttest/virsh.py

```diff
@@ -3052,7 +3052,10 @@ def snapshot_create_as(name, options="", **dargs):
     if options is not None:
         cmd += " %s" % options
 
-    return command(cmd, **dargs)
+    # Avoid snapshot-create-as cmd hang on
+    timeout = dargs.get('timeout', 300)
```

Comment:
I think we could just run with 
```
virsh.snapshot_create(vm_name, options, timeout=300)
```
in tp-libvirt, it should work.

Reply:
You mean virsh.snapshot_create_as(vm_name, options, timeout=300) right?
 
Of course that should work. 
But i think set default timeout inside func snapshot_create_as is much better,
Reason
1. Many jobs have already introduced snapshot_create_as with setting timeout option
2. When devlopers don't set timeout option these cases which call snapshot_create_as may be TIMEOUT in CI env.
 
To avoid this situation, i think a default timeout inside snap_create_as is needed or make timeout option a must.

Reply:
I'm not sure about that. My opinion is we shouldn't set a fixed timeout just because our ci is limiting time of a case. Sometimes the image could be big, or the host performance is not so good, that could cause the execution time longer than usual. 
Anyway, I'll ask @kylazhang 's opinion too.

Reply:
@liuyd96 I agree with @chloerh . I think timed out issue in CI may caused by several reasons, ci timeout setting, case setting or product issue or a performance issue as @chloerh mentioned. As a common func, I think we shouldn't set it.

Reply:
Ok, i will close this PR and try other solutions.
Thanks.